spec/tail_spec.rb

```diff
@@ -146,6 +147,7 @@
     let(:directory) { Stud::Temporary.directory }
     let(:file_path) { File.join(directory, "1.log") }
     let(:position) { :beginning }
+    let(:should_yield) { [[file_path, "line1"], [file_path, "line2"]] }
```

Comment:
this is one of those DRY actions that I believe hurt readability in favor of conciseness.
reading the test I'd rather have the assertion in front of me instead of having to scroll up.
what do you think?